merge/update.go

```diff
@@ -62,19 +62,13 @@ func (s *Updater) update(oldObject, newObject *typed.TypedValue, version fieldpa
 			var err error
 			versionedOldObject, err := s.Converter.Convert(oldObject, managerSet.APIVersion())
 			if err != nil {
-				if s.Converter.IsMissingVersionError(err) {
-					delete(managers, manager)
-					continue
-				}
-				return nil, nil, fmt.Errorf("failed to convert old object: %v", err)
```

Comment:
Did i remember correctly that we explicitly didn't want to fail but drop?

Reply:
Correct, I think the new code is right.

Reply:
After the meeting yesterday, do we still want this behavior (at least for apply).